src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      class="searchbar"
+      method="post"
+      data-qa="big"
+    >
```

Comment:
Go through the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and fix your code style, look closely at the 3rd item